readme.md

```diff
@@ -131,7 +137,7 @@ $ rails g migration create_artists
 This creates a migration file `db/migrate/20150726145027_create_artists.rb`. Let's see what it looks like...  
 
 The numbers at the beginning of a migration file are a timestamp.
-* Rails uses this timestamp to determine which migration should be run and in what order, so if you're copying a migration from another application or generate a file yourself, be aware of its position in the order.  
+* Rails uses this timestamp to determine which migration should be run and in what order. If you're copying a migration from another application or generate a file yourself, be aware of its position in the order.  
 
 > [Source](http://edgeguides.rubyonrails.org/active_record_migrations.html#creating-a-migration)  
 
```

Comment:
I don't like the word `Source` just hanging out there like that. Can we embed in the text?